Makefile

```diff
@@ -64,7 +64,7 @@ typecheck: typecheck-js typecheck-python
 typecheck-js:
 	@yarn tsc
 typecheck-python:
-	@yarn pyright
+	@yarn pyright --skipunannotated
```

Comment:
To clarify in case there are any questions, this flag is set so that the `prairielearn.py` file can have typechecking enabled while not having to fix all of the type issues in that file. Annoyingly, this flag cannot be set in the config file.